.github/scripts/trymerge.py

```diff
@@ -1618,28 +1626,43 @@ def remove_job_name_suffix(name: str, replacement: str = ")") -> str:
 
 
 def is_broken_trunk(
-    name: str,
+    check: Any,
     drci_classifications: Any,
 ) -> bool:
     if not name or not drci_classifications:
         return False
 
+    name = check.name
+    job_id = check.job_id
+    workflow_id = check.workflow_id
+
     # Consult the list of broken trunk failures from Dr.CI
     return any(
-        name == broken_trunk["name"]
+        (name == broken_trunk["name"])
+        or (job_id == broken_trunk["id"] and workflow_id == broken_trunk["workflowId"])
```

Comment:
Do we need to worry about if job_id/workflow_id is None?

Reply:
Hmm, I think I should add a check to avoid matching 2 None job IDs, it'. might not be possible, but let's play it safe